tick/solver/build/__init__.py

```diff
@@ -1 +1,19 @@
 # License: BSD 3 clause
+
```

Comment:
Maybe could we add a comment telling why we think this is necessary here.

Reply:
yes